README.md

```diff
@@ -80,29 +70,22 @@ mkdir -p /Users/user/Data/unchained
 
 **Note:** Adjust these paths appropriately for your machine.
 
-Next, create a file called `docker-compose.local.yml` in the current folder. See the [docker-compose.local.example](docker-compose.local.example) for more information.
+3. Replace the `source` paths in `docker-compose.yml` with the paths you just made in step 2.
 
-```[shell]
-services:
-  core:
+```yml
+...
     volumes:
       - type: bind
-        source: /Users/user/Data/docker/cache
+        source: /Users/user/Data/cache       <--- REPLACE THIS WITH YOUR PATH
```

Comment:
I think I'd prefer to keep the word `docker` here. I'm speaking just for myself, but I use the `~/Data` folder for many things, not just docker, so removing this forces the appearance of folders at the top level. I would be annoyed by that.

Maybe change the words to `REPLACE THIS WITH ANY VALID PATH`

Also, can you make the `<--` part of the line a comment. In that case, if the user doesn't remove it, it wont' cause problems.